.azure-pipelines/mgmt.yml

```diff
@@ -33,7 +33,7 @@ jobs:
       - task: MSBuild@1
         inputs:
           solution: build.proj
-          msbuildArguments: /t:RunTests /clp:ShowTimestamp $(LoggingArgs) /p:IgnorePathTokens="$(IgnoreBuildTokens)" /p:UseLegacyDirStructure=true /p:SkipTests=true
+          msbuildArguments: /t:RunTests /clp:ShowTimestamp $(LoggingArgs) /p:IgnorePathTokens="$(IgnoreBuildTokens) /p:SkipTests=true"
```

Comment:
Looks like the quoting is off here so SkipTests didn't correctly get set. Can I suggest this PR just undo all changes to the yml and just have the parallel stuff and then we can iterate on the yml and the rest of the test failures.